.do/values.yaml

```diff
@@ -174,3 +176,32 @@ crow:
         context: "/monitoring"
     passwordAuth: {}
   crowdb: {}
+
+rabbitmq-ha:
+  metrics:
+    enabled: true
+    serviceMonitor:
+      enabled: true
+    prometheusRule:
+      enabled: true
+      rules:
+        - alert: Rabbitmq Unavailable
+          expr: |
+            kube_statefulset_status_replicas_ready{statefulset="{{ template "rabbitmq.fullname" . }}"} < kube_statefulset_status_replicas{statefulset="{{ template "rabbitmq.fullname" . }}"}
+          for: 15m
+          labels:
+            severity: warning
+          annotations:
+            summary: Insufficient Rabbitmq Nodes
+            description: |
+              Insufficient Rabbitmq Nodes - {{ "{{ $value }}" }}
```

Comment:
This should alert us when a pod takes too long to transition from  `pending` to `running` state, typically because of the `Waiting for mnesia tables` issue.